packages/core/lib/services/dids/index.test.js

```diff
@@ -0,0 +1,84 @@
+import {assertRpcService} from '../test-utils';
+import {DIDServiceRPC} from './service-rpc';
+import {didService as service} from './service';
+import {validation} from './config';
+import {DIDKeyManager} from '@docknetwork/wallet-sdk-dids';
+
+describe('DID Service', () => {
+  it('ServiceRpc', () => {
+    assertRpcService(DIDServiceRPC, service, validation);
+  });
+
+  it('keypairToDidKeyDocument', async () => {
+    const keypairDoc = {
+      '@context': ['https://w3id.org/wallet/v1'],
+      id: 'urn:uuid:e8fc7810-9524-11ea-bb37-0242ac130002',
+      name: 'My Test Key 2',
+      image: 'https://via.placeholder.com/150',
+      description: 'For testing only, totally compromised.',
+      tags: ['professional', 'organization', 'compromised'],
+      correlation: ['4058a72a-9523-11ea-bb37-0242ac130002'],
+      controller: 'did:key:z6MkjjCpsoQrwnEmqHzLdxWowXk5gjbwor4urC1RPDmGeV8r',
+      type: 'Ed25519VerificationKey2018',
+      privateKeyBase58:
+        '3CQCBKF3Mf1tU5q1FLpHpbxYrNYxLiZk4adDtfyPEfc39Wk6gsTb2qoc1ZtpqzJYdM1rG4gpaD3ZVKdkiDrkLF1p',
+      publicKeyBase58: '6GwnHZARcEkJio9dxPYy6SC5sAL6PxpZAB6VYwoFjGMU',
+    };
+    const res = await service.keypairToDidKeyDocument({keypairDoc});
+
+    expect(DIDKeyManager.keypairToDidKeyDocument).toBeCalledWith(keypairDoc);
+    expect(res).toBeDefined();
+    expect(res).toHaveProperty('didDocument');
+    expect(res).toHaveProperty('keyDoc');
+  });
+
+  it('getDidResolution', async () => {
```

Comment:
what are we expecting this test to prove?

Reply:
might need some help from @osinakayah here

Reply:
I've renamed the test case